.gitignore

```diff
@@ -1,4 +1,5 @@
-dist/
+# Removed dist from gitignore so package could be installed from repo
```

Comment:
I'm against the idea to removing 'dist' from gitignore since it would increase the repo size dramatically with unreadable but auto-generatable code. Also, because not everyone would remember to build the code and add 'dist' to git before he/she committing a change, the original intent of 'package could be installed from repo' could be a bubble. And it would further cause a mismatch between 'dist' folder and 'src' folder.

Installing from repo instead of npm registry is definitely necessary under very rare circumstances. I usually have to do so when there are some critical updates, in the repo, that I need but they're not yet released. Then I have several ways to get the updates:
1. Clone the repo locally and build it locally. Then configure $PATH to correctly locate the locally built version.
    This approach works nice when the target module is some executable tools, like cli, transpiler, etc, but not very usefule when it is a library dependency of a project, which is often  under the hands of multiple team members thus maintaing a dependency locally on every machine is not desirable.
2. Fork the repo, and make a commit with 'dist' code (added to git by `git add -f`, overriding .gitignore temporarily), and modify the `package.json` dependency to the forked repo. Since installing dependency from a remote repo is always a temporary fix (we would update it to npm registry once the critical updates get released), pointing to the main repo or a fored repo makes no difference.